test/e2e/handler/nnce_conditions_test.go

```diff
@@ -128,23 +181,36 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Type:   shared.NodeNetworkConfigurationEnactmentConditionMatching,
 					Status: corev1.ConditionTrue,
 				},
-			}
-			for _, node := range nodes {
-				By(fmt.Sprintf("Check %s failing state is reached", node))
-				enactmentConditionsStatusEventually(node).Should(ConsistOf(failingEnactmentConditions...), "should eventually reach failing conditions at enactments")
+				shared.Condition{
+					Type:   shared.NodeNetworkConfigurationEnactmentConditionAborted,
+					Status: corev1.ConditionTrue,
+				},
 			}
 			By("Check policy is at degraded state")
 			waitForDegradedTestPolicy()
 
-			By("Check that the enactment stays in failing state")
+			for _, node := range nodes {
+				By(fmt.Sprintf("Check %s failing or aborted state is reached", node))
+				enactmentConditionsStatusEventually(node).Should(
+					SatisfyAny(
+						ConsistOf(failingEnactmentConditions...),
+						ConsistOf(abortedEnactmentConditions...),
+					), "should eventually reach failing of aborted conditions at enactments")
+			}
+
+			By("Check that the enactment stays in failing or aborted state")
 			var wg sync.WaitGroup
 			wg.Add(len(nodes))
 			for i, _ := range nodes {
 				node := nodes[i]
 				go func() {
 					defer wg.Done()
 					By(fmt.Sprintf("Check %s failing state is kept", node))
-					enactmentConditionsStatusConsistently(node).Should(ConsistOf(failingEnactmentConditions...), "should consistently keep failing conditions at enactments")
+					enactmentConditionsStatusConsistently(node).Should(
+						SatisfyAny(
+							ConsistOf(failingEnactmentConditions...),
+							ConsistOf(abortedEnactmentConditions...),
+						), "should consistently keep failing or aborted conditions at enactments")
```

Comment:
ditto

Reply:
I think in this case, we can keep the or, because should any of those enactment change, it would need to be at least in `matching` or `notMatching`.
Afterwards I again check the numbers of aborted and failing